### PR TITLE
A few more things for the Psyche locker.

### PR DESCRIPTION
**Pull request by Birdtalon**
A request from Piccione🕊on discord.

A few more things for the Psyche locker.

- 1 Extra Haloperidol pill, 1 extra Methamphetamine pill, 2 Hydrocodone pills
- 1 Random plushie.
- 3 Random chocolate candies.

🆑 
tweak: A few more items in the Psyche locker.
/🆑

### CODE REVIEW

**Comment by StreakyHaddock** [comment]
Now show me on the doll where he robusted you.

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm [hunk 1]
```diff
@@ -130,6 +137,12 @@
 	new /obj/item/weapon/reagent_containers/syringe(src)
 	new /obj/item/weapon/reagent_containers/glass/bottle/ether(src)
 	new /obj/item/weapon/storage/pill_bottle/psychiatrist(src)
+	new /obj/random/plushie(src)
+	var/i = 0
+	while(i < 3)
```
Comment (Fox-McCloud) [inline code/game/objects/structures/crates_lockers/closets/secure/medical.dm]: gross.

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm [hunk 2]
```diff
@@ -130,7 +137,10 @@
 	new /obj/item/weapon/reagent_containers/syringe(src)
 	new /obj/item/weapon/reagent_containers/glass/bottle/ether(src)
 	new /obj/item/weapon/storage/pill_bottle/psychiatrist(src)
-
+	new /obj/random/plushie(src)
+	for(var/i = 0, i < 3, i++)
```
Comment (tigercat2000) [inline code/game/objects/structures/crates_lockers/closets/secure/medical.dm]: for(var/i in 0 to 3)

Comment (Fox-McCloud) [reply code/game/objects/structures/crates_lockers/closets/secure/medical.dm]: I'd recommend `1 to 4`, personally, as the standard is starting with `1` as opposed to `0`.

**Comment by MarcellusPye** [comment]
Please update the in game changlog too

**Pull request merged**